utils.groovy

```diff
@@ -199,4 +199,18 @@ def get_push_trigger() {
     ]
 }
 
+def get_artifacts(pipecfg, stream, basearch) {
+    // Get artifacts from config.yaml
+    def artifacts
+    if  (pipecfg.streams."${stream}".artifacts?."${basearch}" != null) {
+        artifacts = pipecfg.streams."${stream}".artifacts."${basearch}"
+        return artifact
+    }
+    else { // Merge default with additional artifacts
+        def default_artifacts =  pipecfg.default_artifacts."${basearch}"
+        def additional_atifacts = pipecfg.streams."${stream}".additional_artifacts?."${basearch}"
+        artifacts = default_artifacts + additional_atifacts
+    }
+    return artifacts
```

Comment:
Let's use [`artifacts.unique()`](https://docs.groovy-lang.org/latest/html/groovy-jdk/java/util/List.html#unique()) here to be defensive against duplicates?